### DISCUSSION TITLE
Scrum Prep Milestone 2-2

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#109] [General]
Each team member will comment progress made through the past week.

**Comment by JuanIranzo** [comment]
During this week, I worked on creating several tasks for Database team members. I was also assigned to the task of creating the Sell model for the database, which I worked on and completed.

**Comment by JadStelar** [comment]
I have worked on a task on researching and creating a summary of the setup and usage of Pass, a password manager, to be used later for the implementation phase.
-Jahdiel Montero

**Comment by ActuallySeb** [comment]
Hell week, not much other than research was accomplished.

**Comment by CovasJJ** [comment]
During this week I wasn't able to do much work because I had many exams.

**Comment by cristian-barreras** [comment]
I have been working on the reCAPTCHA task. I have already created the necessary Vue, HTML, and JS files for the reCAPTCHA. I am still going through iterations for the layout of the Vue page and working on the functionality of the reCAPTCHA.

**Comment by AlvinBadillo** [comment]
Due to other course work and the two exams I had this week, I was unable to make significant contributions this week.

**Comment by franrov4** [comment]
Met with managers and team leads this week, discussed project progress, as well as next steps for the milestone. Made progress establishing a connection from the database with the backend. Next steps: testing data flow and ensuring stable connection.

**Comment by YampierFernandez** [comment]
This week I participated in a meeting between team leads and managers, based on what was said in the meeting I started a discussion for the database design team and created a task.

**Comment by gcal77** [comment]
This week I created multiple tasks for the second milestone and was assigned them. One of them involved creating a draft for a better client login page UI design.

-Gabriel Calvente

**Comment by JuanDelPueblo** [comment]
This week I was unable to do any work due to my laptop's display failing and having no spare computer to develop on.

**Comment by Christian2101** [comment]
This week has been a hellweek due to having many exams I was unable to work on the documentation. Will begin to work on fixing the documentations with the feedback given by the professor.

**Comment by Kaariinaa08** [comment]
This week, I created a task to develop the product listing search page based on the Sketch design. I will implement a responsive grid and list layout, a filter sidebar, pagination, and product sorting, ensuring all components match the design's styling and functionality.

- Karina Lopez Rodriguez

**Comment by PerezMarcelo** [comment]
I attended a team leads meeting with the managers and I am currently working on a captcha task.

**Comment by elliot-gomez** [comment]
This week, I primarily focused on reviewing tasks and goals due to midterms for all my classes. Unfortunately, I didn’t have much time to work on other projects. Additionally, I faced some challenges with the power going out at my apartment, which limited my ability to complete any significant work. However, I utilized the time I had to ensure I’m aligned with the team’s objectives and prepared for upcoming tasks once my schedule stabilizes.

- Elliot J. Gomez Coriano

**Comment by JoshDG03** [comment]
I worked on the sell controller for the data base, I was able to implement all CRUD operations
